boards/arm/stm32l562e_dk/stm32l562e_dk.dts

```diff
@@ -48,3 +48,8 @@
 &iwdg {
 	status = "okay";
 };
+
```

Comment:
like for nucleo_l552, I think this should be added to common file stm32l562e_dk_common.dtsi instead of this file (secure file) and then both yaml file should reflect this change.